github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -278,6 +309,38 @@ postsubmits:
               limits:
                 memory: "16Gi"
                 cpu: "500m"
+    - name: publish-vm-image-builder-image
+      always_run: false
+      run_if_changed: "images/vm-image-builder/.*"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: phx-prow
```

Comment:
Do we want to use this cluster for new jobs? I thought this would be turned off at some point?

Reply:
Good point, I copied it from the old job to publish bootstrap + golang images. It should be ok to keep using phx-prow for these small jobs, the cluster is still up and I haven't heard that it is going to be decommissioned any time soon. We should definitely stop scheduling e2e jobs there given that the bare metals were removed though.